kie-server-parent/kie-server-controller/kie-server-controller-standalone/pom.xml

```diff
@@ -66,15 +76,87 @@
       <artifactId>cdi-api</artifactId>
       <scope>provided</scope>
     </dependency>
-    
+
     <dependency>
       <groupId>org.jboss.spec.javax.servlet</groupId>
       <artifactId>jboss-servlet-api_3.1_spec</artifactId>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
+      <groupId>io.swagger</groupId>
+      <artifactId>swagger-jaxrs</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>javax.ws.rs</groupId>
+          <artifactId>jsr311-api</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+
+    <dependency>
+      <groupId>io.swagger</groupId>
+      <artifactId>swagger-core</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>io.swagger</groupId>
+      <artifactId>swagger-annotations</artifactId>
+    </dependency>
+
   </dependencies>
   <build>
     <finalName>kie-server-controller-standalone-${project.version}</finalName>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.codehaus.cargo</groupId>
+          <artifactId>cargo-maven2-plugin</artifactId>
```

Comment:
Since this is an assembly module, is the cargo needed here? If not, then we don't need `<version.tomcat9>9.0.12</version.tomcat9>` as well.

Reply:
I added Cargo in here, just as a way to quickly test/run the assembly to check my changes. If you think there is a better way for doing that, I'm open to suggestions.

Reply:
I just wanted to know if we want to leave it for future testing. If yes, I am ok with it.